op-e2e/system/proofs/build_helper.go

```diff
@@ -1,4 +1,4 @@
-package op_e2e
+package proofs
```

Comment:
The `../op-program` in this file will need to be adapted for the deeper directory structure its in now.